admin_manual/enterprise/firewall/file_firewall.rst

```diff
@@ -132,6 +132,29 @@ Block by System Tag::
 .. figure:: images/firewall-3.png
    :alt: Protecting files tagged with "Confidential" from outside access
 
+Logging
+~~~~~~~
+
+Firewall logging can be set to **Off**, **Blocked Requests Only** or **All Requests**
+
+Off
+^^^
+
+The firewall blocks according to the defined rules but does not log any of its actions.
+
+Blocked Requests Only
+^^^^^^^^^^^^^^^^^^^^^
+
+The firewall logs blocked requests to the system log at **warning** level. To see these logs, the system log level must be
+set to include **warning**.
```

Comment:
Regarding "set to include **warning**", should it not be "set to a minimum level of **warning**"?

Reply:
That is the convention we have used elsewhere - I will fix it.
I find the use of "minimum" and "maximum" a bit confusing in these contexts.
The amount of log entries goes from minimum error-warning-info-debug to maximum.
But the log level number that you put in config.php is in the reverse; 0 (the minimum number) is debug (the maximum amount of logging).